pages/datasets/scaffoldviewer/_id.vue

```diff
@@ -51,8 +51,9 @@
               :state="state"
               :display-markers="displayMarkers"
               :url="scaffoldUrl"
-              :traditional="traditional"
               :background-toggle="backgroundToggle"
+              :region="region"
+              :viewURL="viewURL"
```

Comment:
I would have this as `:view-url="viewURL"`

Reply:
This would require a change on the dependency. I will derfer doing this until I make a clean up release of the scaffoldvuer dependency.